staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/testcase.go

```diff
@@ -727,6 +752,37 @@ func NewMutatingTestCases(url *url.URL) []MutatingTest {
 			ExpectLabels:      map[string]string{"pod.name": "my-pod"},
 			ExpectAnnotations: map[string]string{"removelabel.example.com/key1": "value1"},
 		},
+		{
+			Name: "first webhook remove labels from CRD, second webhook shouldn't be called",
+			Webhooks: []registrationv1beta1.MutatingWebhook{{
+				Name:              "removelabel.example.com",
+				ClientConfig:      ccfgSVC("removeLabel"),
+				Rules:             matchEverythingRules,
+				NamespaceSelector: &metav1.LabelSelector{},
+				ObjectSelector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"remove": "me",
+					},
+				},
+				AdmissionReviewVersions: []string{"v1beta1"},
+			}, {
+				Name:              "shouldNotBeCalled",
```

Comment:
Include one test where the selector does match?

Reply:
the first webhook in this set is called